web/app.js

```diff
@@ -402,7 +402,7 @@ const PDFViewerApplication = {
     } else {
       eventBus = new EventBus();
     }
-    this.eventBus = eventBus;
+    this.eventBus = this.preferences.eventBus = eventBus;
```

Comment:
Since the functionality seems to be for Firefox only, should we have the `this.preferences.eventBus = eventBus;` assignment only in that case by placing it in the MOZCENTRAL branch just above?